subprojects/common/report-viewer/src/main/kotlin/com/avito/report/ReportsApi.kt

```diff
@@ -66,15 +67,21 @@ interface ReportsApi : ReportsAddApi, ReportsFetchApi {
             verboseHttp: Boolean = false
         ): ReportsApi {
 
+            val logger = loggerFactory.create<ReportsApi>()
+
             return ReportsApiImpl(
                 loggerFactory = loggerFactory,
                 requestProvider = JsonRpcRequestProvider(
                     host = host,
                     httpClient = getHttpClient(
                         verbose = verboseHttp,
-                        logger = loggerFactory.create<ReportsApi>(),
+                        logger = logger,
                         readTimeoutSec = readTimeout,
-                        writeTimeoutSec = writeTimeout
+                        writeTimeoutSec = writeTimeout,
+                        retryInterceptor = RetryInterceptor(
+                            logger = logger,
+                            allowedMethods = listOf("POST")
+                        )
```

Comment:
What statuses did we receive from RV?
Do we need to add them to the interceptor?

Reply:
https://github.com/avito-tech/avito-android/blob/0bada247948b9db5ff93a89b33251b730ab65bcb/subprojects/common/okhttp/src/main/kotlin/com/avito/http/RetryInterceptor.kt#L18-L22

These defaults are ok, it's 502 mainly